master/buildbot/buildslave/__init__.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, name, password, max_builds=None,
         self.password = password
         self.friendly_name = friendlyName
         self.eid = eid  # External ID
+        self.fqdn = fqdn
```

Comment:
Thanks, good to know.  FQDN is descriptive, but apparently, fqdn is not :) So better document all new parameters anyway.